docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  ganache-cli:
+    image: trufflesuite/ganache-cli
+    container_name: ganache-cli
+    ports:
+      - "8145:8545"
+    #command: ["--accounts=100", "--hostname=0.0.0.0", "--gasLimit=100000000", "--defaultBalanceEther=1000000", "--mnemonic=xxx", "--networkId=99999"]
```

Comment:
I'm fine with adding docker to this repo, but can you clean up these unused lines please?